### PR TITLE
Added group label key to fix label info onDrag

### PR DESCRIPTION
**Pull request by thiagosatoshi**
**Issue Number**

https://github.com/namespace-ee/react-calendar-timeline/issues/442#issue-367851639

**Overview of PR**

The fix involves introducing a new key in the Timeline keys object called `groupLabelKey`, which is defaulted to `title`. This will preserve existing functionality. When the `groupLabelKey` is specified to point to a string attribute of the group object, that is used to render the text in the InfoLabel.

This allows the developer to split what is used to render the Sidebar with what is used to render the InfoLabel.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-calendar-timeline",
-  "version": "0.20.0",
```
Comment (Ilaiwi) [inline package.json]: this is a new feature so it would be 0.21.0

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/namespace-ee/react-calendar-timeline",
   "repository": {
     "type": "git",
-    "url": "https://github.com/namespace-ee/react-calendar-timeline.git"
```
Comment (Ilaiwi) [inline package.json]: could you please revert this?

**Comment by Ilaiwi** [comment]
thank you for this. small changes need to be done but everything else is fine

**Comment by thiagosatoshi** [comment]
@Ilaiwi thanks for the review, I have now updated the PR as requested. Please have a look again once you have time. 👍

**Comment by thiagosatoshi** [comment]
@Ilaiwi done. Please let me know if that is OK now?

**Comment by Ilaiwi** [comment]
@thiagosatoshi this will be released in 0.22.0 during the next two weeks thank you for your work

**Pull request merged**